From 5ec8c855f307a89f08013c832bd7e121c8f83327 Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Thu, 15 Dec 2016 21:47:28 -0500
Subject: [PATCH 33/55] Fix some test cases where the resulting default index
 is wrong

Fix two issues with tests where removing a kernel or adding a boot entry
at a user specified index had the wrong default index selected in the
expected output.

One of these issues is the --set-index test cases for grub 1, which
expect --make-default is implied where it should not be.

The third is a removal test, where the default index in the result data
is off by one when we've removed 2 entries keyed off of the kernel
filesystem path.

This change causes these test cases to fail.

Resolves: rhbz#1285601
---
 test/results/add/g1.17   | 2 +-
 test/results/add/g1.9    | 2 +-
 test/results/remove/g7.1 | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/results/add/g1.17 b/test/results/add/g1.17
index 6a388228768..eda4e93329d 100644
--- a/test/results/add/g1.17
+++ b/test/results/add/g1.17
@@ -7,7 +7,7 @@
 #          kernel /vmlinuz-version ro root=/dev/sda1
 #          initrd /initrd-version.img
 #boot=/dev/hda
-default=1
+default=0
 timeout=10
 splashimage=(hd0,0)/grub/splash.xpm.gz
 title Red Hat Linux (2.4.7-2)
diff --git a/test/results/add/g1.9 b/test/results/add/g1.9
index 310623d13d2..8976e762d74 100644
--- a/test/results/add/g1.9
+++ b/test/results/add/g1.9
@@ -7,7 +7,7 @@
 #          kernel /vmlinuz-version ro root=/dev/sda1
 #          initrd /initrd-version.img
 #boot=/dev/hda
-default=2
+default=1
 timeout=10
 splashimage=(hd0,0)/grub/splash.xpm.gz
 title Red Hat Linux (2.4.7-2)
diff --git a/test/results/remove/g7.1 b/test/results/remove/g7.1
index ea1a6fbaec9..611d7a16685 100644
--- a/test/results/remove/g7.1
+++ b/test/results/remove/g7.1
@@ -1,4 +1,4 @@
-default=1
+default=0
 timeout=10
 splashimage=(hd0,5)/boot/grub/splash.xpm.gz
 
-- 
2.17.1

